packages/react-components/react-swatch-picker-preview/src/components/SwatchPicker/useSwatchPickerStyles.styles.ts

```diff
@@ -6,12 +6,6 @@ export const swatchPickerClassNames: SlotClassNames<SwatchPickerSlots> = {
   root: 'fui-SwatchPicker',
 };
 
-export const swatchPickerCSSVars = {
```

Comment:
can you explain why reason of these changes ?

Reply:
it was not required to use dynamic style for `spacing`, I moved those to styles and spacing is taken from the state